assets/js/plausible.rb

```diff
@@ -0,0 +1,10 @@
+class Plausible
+    def initialize(config)
+        @domain = config['domain']
+        @source = config['source']
+    end
+
+    def render()
+        return "<script async defer data-domain=\"#{@domain}\" src=\"#{@source}\"></script>"
```

Comment:
Plausible.js is mentioned in the _config.yml file as being served by us, but I don't see the javascript file actually in our assets. Or, does this generate the javascript file somehow?

Reply:
This PR should be adding plausible.js to assets/js/

Reply:
Right now, it looks like the source is coming from us per the config file, but I don't see anything in the repo that would actually host that file. If we merge this PR, we may end up getting a 404. If we do get a 404, try changing the source in the config file back to plausible and see if that fixes it.